standard_plus_issue/data/template_email_standard_plus_issue.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+  <data noupdate="1">
+
+    <!--Email template -->
+    <record id="email_template_standard_plus_issue" model="mail.template">
+      <field name="email_from">${(object.user_id.email or object.user_id.company_id.email)|safe}</field>
+      <field name="subject">[${object.user_id.company_id.name}] | ${object.name} | Request: ${object.type} | Priority: ${object.priority}</field>
+      <field name="model_id" ref="standard_plus_issue.model_standard_plus_issue"/>
+      <field name="auto_delete" eval="True"/>
+      <field name="body_html"><![CDATA[
+        <h3>Request ${object.name}</h3>
+        <p>
+          A request of type <strong>${object.type}</strong> for project ${object.user_id.company_id.name} has been submitted by ${(object.user_id.name)} with a level '${object.priority}' priority.
+        </p>
+        <p>
+          <strong>Concerned module:</strong>
+          % if object.module_id.name:
+            ${object.module_id.name}
+          % else:
+            left blank
```

Comment:
It literally prints left blank. (eg: Concerned module: left blank). It indicates that the user didn't fill the field.